src/redux/rockets/reservedRockets.js

```diff
@@ -0,0 +1,20 @@
+const GET_RESERVED_ROCKETS = 'GET_RESERVED_ROCKETS';
+
+export const loadReservedRockets = (payload) => (dispatch) => {
+  dispatch({
+    type: GET_RESERVED_ROCKETS,
+    payload,
+  });
+};
+
+const reservedRocketsReducer = (state = [], action = { type: 'default' }) => {
+  switch (action.type) {
+    case GET_RESERVED_ROCKETS:
+      return [...action.payload.filter((rocket) => rocket.reserved)];
+
+    default:
+      return state;
+  }
+};
+
+export default reservedRocketsReducer;
```

Comment:
- [ ] [Optional] It would be nice to add this to your other slice file that contains the `rocketsReducer`. This would make your code cleaner, avoid duplication and it would be easy to maintain because you know where to easily trace things in case you want to add a feature or you want to fix a bug. 👍